core/src/main/java/tc/oc/pgm/command/parsers/PartyParser.java

```diff
@@ -43,9 +43,8 @@ public ArgumentParseResult<Party> parse(
   @Override
   public @NotNull List<@NotNull String> stringSuggestions(
       @NotNull CommandContext<CommandSender> context, @NotNull CommandInput input) {
-    final String next = input.peekString();
     List<String> teams = teamParser.stringSuggestions(context, input);
-    if (LiquidMetal.match("obs", next)) teams.add("obs");
+    if (LiquidMetal.match("obs", input.peekString())) teams.add("obs");
```

Comment:
can't guarantee this either, you need to grab the state before it potentially gets manipulated by downstream